silkworm/rpc/ethbackend/remote_backend.cpp

```diff
@@ -202,6 +202,17 @@ Task<evmc::bytes32> RemoteBackEnd::get_block_hash_from_block_number(uint64_t num
     co_return hash;
 }
 
+Task<std::string> RemoteBackEnd::canonical_body_for_storage(uint64_t number) {
+    const auto start_time = clock_time::now();
+    UnaryRpc<&::remote::ETHBACKEND::StubInterface::AsyncCanonicalBodyForStorage> canonical_body_for_storage_rpc{*stub_, grpc_context_};
+    ::remote::CanonicalBodyForStorageRequest request;
+    request.set_blocknumber(number);
+    const auto reply = co_await canonical_body_for_storage_rpc.finish_on(executor_, request);
+    SILK_TRACE << "RemoteBackEnd::canonical_body_for_storage bn="
```

Comment:
Add missing `<< block_number` at the end of this line

Reply:
ok